### PR TITLE
[2.4] PANDARIA: Add v0.1.5002 monitoring chart with gpu monitoring v2

### PR DESCRIPTION
**Pull request by jianghang8421**
https://github.com/cnrancher/pandaria/issues/1498

### CODE REVIEW

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-coredns/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Creates ServiceMonitor CRD of coredns for Kubernetes which maintaining by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: exporter-coredns
+version: 0.0.1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-coredns/Chart.yaml]: remove useless line

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-coredns/templates/servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      action: replace
+      regex: (.+)
+      replacement: $1
+    - sourceLabels:
+      - __meta_kubernetes_pod_node_name
+      targetLabel: node
+      action: replace
+      regex: (.+)
+      replacement: $1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-coredns/templates/servicemonitor.yaml]: remove useless line

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-controller-manager/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Creates ServiceMonitor CRD of controller manager for Kubernetes which maintaining by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: exporter-kube-controller-manager
+version: 0.0.1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-controller-manager/Chart.yaml]: remove useless line

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-dns/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Creates ServiceMonitor CRD of kube-dns for Kubernetes which maintaining by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: exporter-kube-dns
+version: 0.0.1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-dns/Chart.yaml]: remove useless line

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-etcd/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Creates ServiceMonitor CRD of etcd for Kubernetes which maintaining by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: exporter-kube-etcd
+version: 0.0.1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-etcd/Chart.yaml]: remove useless line

File: charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-scheduler/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Creates ServiceMonitor CRD of scheduler for Kubernetes which maintaining by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: exporter-kube-scheduler
+version: 0.0.1
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.1.5002/charts/exporter-kube-scheduler/Chart.yaml]: remove useless line

**Review by JacieChao — APPROVED** [review]
LGTM

**Pull request merged**